model/src/main/kotlin/Environment.kt

```diff
@@ -52,6 +52,16 @@ data class Environment(
     companion object {
         val ORT_VERSION = this::class.java.getResource("/VERSION").readText()
 
-        private val RELEVANT_VARIABLES = listOf("COMSPEC", "OS", "OSTYPE", "SHELL")
+        private val RELEVANT_VARIABLES = listOf(
+                // Windows variables.
+                "OS",
+                "COMSPEC",
+                // Unix variables.
+                "SHELL",
+                "TERM",
+                // General variables.
+                "GOPATH",
+                "JAVA_HOME"
+        )
```

Comment:
OSTYPE is set on my machine (Ubuntu 18.04 -> "linux-gnu"). Some other candidates for the list: "ANDROID_HOME", "LANG", "HOSTTYPE" ("x86_64")

Reply:
All done except for `LANG`.